lib/web_ui/dev/integration_tests_manager.dart

```diff
@@ -440,3 +633,30 @@ const Map<String, List<String>> blockedTestsListsMap = <String, List<String>>{
     'target_platform_ios_integration.dart',
   ],
 };
+
+/// Tests blocked for one of the build modes.
+///
+/// If a test is not suppose to run for one of the modes also add that test
+/// to the corresponding list.
+// TODO(nurhan): Remove the failing test after fixing.
+const Map<String, List<String>> blockedTestsListsMapForModes =
```

Comment:
It's still weird that one needs to recompile `felt` in order to change the list of tests they want to run. I'd prefer a YAML file that lives closer to the test code itself.

Reply:
Since it's been a while we added blocked tests logic, it seems like the request is outside of this PR. I create another issue and address that change there.